app/api/currentuser/join_group_by_code.feature

```diff
@@ -87,6 +111,10 @@ Feature: Join a group using a code (groupsJoinByCode)
       | 14                | 14             | 1       |
       | 15                | 15             | 1       |
       | 21                | 21             | 1       |
+    And the table "attempts" should be:
```

Comment:
I am not sure it is worth checking it in every scenario. It makes tests more difficult to maintain.

Reply:
I cannot agree here. I added the checks before modifying the services and that helped me to be sure everything is done. If we ever modify the services I would want the checks to keep passing. So I think they will help us in the future as well.

Reply:
What I meant is that it should be done for every service, but maybe not for every scenario (their change between scenario is not affecting attempts)
But I am fine with it if you are convinced this is better.